en/middleware/micrortps.md

```diff
@@ -98,9 +126,15 @@ The *Client* application can be launched from [NuttShell/System Console](../debu
   -s <sending port>       UDP port for sending. Default 2020
 ```
 
-> **Note** By default the *Client* runs as a daemon, but you will need to start it manually. The PX4 Firmware initialisation code may in future automatically start the *Client* as a permanent daemon process.<!-- at that point, most of this section would move into the "manual generation" doc: https://github.com/PX4/Firmware/pull/7663#issuecomment-317928506 -->
+> **Note** By default the *Client* runs as a daemon, but you will need to start it manually. The PX4 Firmware initialization code may in future automatically start the *Client* as a permanent daemon process.
 
-## Agent (Off Board FastRTPS Interface)
+For example, in order to run the *Client* daemon with SITL, and since the data is being shared through UDP, it's required that the daemon is started using the UDP as the transport protocol:
+
+```sh
+micrortps_client start -t UDP
```

Comment:
It would be MUCH better if this was set through a user-side parameter.

Reply:
This is not standard yet. This would be also something that you will need to set for each airframe config. For the POSIX case,s that would be using UDP, but for the Nuttx and other targets, it would need to be serial. This is something to address further.